clj-refactor.el

```diff
@@ -3254,6 +3254,17 @@ if REMOVE-PACKAGE_VERSION is t get rid of the (package: 20150828.1048) suffix."
         (replace-regexp-in-string " (.*)" "" version)
       version)))
 
+(defcustom cljr-injected-middleware-version (cljr--version t)
+  "The refactor-nrepl version to be injected.
+
+You can customize this in order to try out new releases.
+
+If customizing it, you most likely should `(setq cljr-suppress-middleware-warnings nil)' too,
+for avoiding a warning that would be irrelevant for this case."
+  :group 'cljr
+  :type 'string
+  :package-version "3.0.0")
```

Comment:
I realised cider.el has `:safe #'<pred>` for all its defcustoms, should I do the same here? Does it make a difference? @bbatsov

Reply:
It's not required but it's useful. Otherwise you'd get warnings when setting something via dir locals.